README.md

```diff
@@ -50,7 +51,8 @@ class Customer(models.Model):
         app_label = 'myapp'
 
 class PreferredCustomer(pg.View):
-    projection = ['myapp.Customer.*']
+    projection = ['myapp.Customer.*',]
+    dependencies = ['myapp.Customer',]
```

Comment:
Would it be possible to get this into the `class Meta` please? Long-term, I'd like to have the meta-properties there.